extras/vfd16lf01/LICENSE

```diff
@@ -0,0 +1,7 @@
+This is a driver for Samsung 16LF01 series VFD.
+
+Originate written by Daniele Napolitano, distributed under GPL.
```

Comment:
Ok but to be honest I don`t get it.

Driver like this is in fact a wrapper on data-sheet of IC. There is no special magic behind of it.
Even if I will build driver from scratch it will looks in the same way (or at least very similar) cause its a way how this display communicate with the world.